jobs/JGLOBAL_ATM_ANALYSIS_INITIALIZE

```diff
@@ -8,19 +8,29 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "atmanlinit" -c "base atmanl atmanlini
 # Set variables used in the script
 ##############################################
 GDATE=$(date +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
+export gPDY=${GDATE:0:8}
+export gcyc=${GDATE:8:2}
```

Comment:
do we need to export these?

Reply:
I simply followed what's done in `JGLOBAL_ATMOS_ANALYSIS`.  I'm fine with removing the export.

Reply:
we shouldn't export variables not needed by child scripts.

Reply:
export removed